lib/data/wrapper.js

```diff
@@ -96,7 +96,7 @@ const data = {
             });
     },
 
-    get: (objectGetInfo, log, cb) => {
+    get: (objectGetInfo, log, params, cb) => {
```

Comment:
it isn't used in s3 at all, but this function has to have the same prototype as a function in Mystique which needs to pass extra params, so that responseStreamData in Arsenal can call either as the 'dataRetrievalFn' without differentiating between the originating project.